spyder/plugins/ipythonconsole.py

```diff
@@ -72,6 +72,9 @@
 dependencies.add("qtconsole", _("Integrate the IPython console"),
                  required_version=QTCONSOLE_REQVER)
 
+IPYTHON_REQVER = ">=4.0"

```

Comment:
This needs to be more fine grained, like this

     IPYTHON_REQVER = ">=4.0;<6.0" if PY2 else ">=4.0"